common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -3010,23 +3102,23 @@ colored_emblem_texture_lists = {
 	}	
 	### Muslims
 	mena_circled_frame_list = {
-		1000 = "ce_tamgha_regular_circle_frame.dds"
-		1000 = "ce_tamgha_doted_circle_frame.dds"	
-		5000 = "ce_octogon_frame.dds"
-		5000 = "ce_polylobe_frame.dds"
+		100 = "ce_tamgha_regular_circle_frame.dds"
```

Comment:
When you change vanilla lines, you add the tag before changes.
```suggestion
		# Warcraft
		100 = "ce_tamgha_regular_circle_frame.dds"
```

Reply:
Reverted changes and moved elunic_coa_gfx to charge.